testing/sdk_plan.py

```diff
@@ -10,9 +10,16 @@
 import retrying
 
 import sdk_cmd
+import sdk_metrics
 
 TIMEOUT_SECONDS = 15 * 60
 SHORT_TIMEOUT_SECONDS = 30
+ALLOWED_FAILURES = 2
```

Comment:
`ALLOWED_FAILURES` is a little bit misleading here, since it controls the allowed INCREASE in task failures while waiting for a plan status to change.

Reply:
True, I forgot about it after introducing the error diff.